students/M_Sunday/lesson03/Mailroom_Part_1.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+from operator import itemgetter
+
+# Lesson 03, Mailroom Part 1 Assignment
+
+
+# ------- Global Assignments -------
+donor_data = [['John', 'Marks', 5000.34, 2500, 1267],
```

Comment:
I would suggest storing donations separately from names in their own sub-list to simplify your calculations and look ups. So the first item in your `donor_data` list would look something like this:
```
['John Marks', [5000.34, 2500, 1267]
```